terraform/deployments/task-runner/integration.backend

```diff
@@ -0,0 +1,5 @@
+bucket  = "govuk-ecs-terraform-integration"
+key     = "projects/rake-task.tfstate.tfstate"
```

Comment:
`rake-task.tfstate.tfstate` looks incorrect. Plus, this cluster is not just for rake tasks, it's for any arbitrary command.

Reply:
+1 - task-runner.tfstate would be the obvious choice.

Reply:
sorry, too much copy, pasting, I will fix